src/connectors/snowflake/trulens/data/sis_dashboard/pages/Records.py

```diff
@@ -0,0 +1,17 @@
+from snowflake.snowpark.context import get_active_session
+from trulens.connectors.snowflake import SnowflakeConnector
+from trulens.core import TruSession
+from trulens.core.experimental import Feature
+from trulens.dashboard.pages.Records import records_main
+
+if __name__ == "__main__":
+    session = get_active_session()
+    # st.write(session)
+
+    tru_session = TruSession(
+        connector=SnowflakeConnector(
+            snowpark_session=session, database_check_revision=False
+        )
+    )
+    tru_session.experimental_enable_feature(Feature.SIS_COMPATIBILITY)
+    records_main()
```

Comment:
given these are all the same aside from the last line. Can we not pass in an argument or something to say which one to use instead of duplicating all of it?

Reply:
Also why is Leaderboard.py now in the `pages` folder?

Reply:
Streamlit creates the app by reading file structure, so we need a separate file for each of the pages

Reply:
Leaderboard is the homepage show shouldn't need to be in /pages iirc

Reply:
can we not just have them all be like
```python
import base_file

base_file.main("leaderboard")
```
and have some `base_file` with the main stuff?